addons/website_sale/static/src/js/website_sale.editor.js

```diff
@@ -588,7 +588,7 @@ options.registry.WebsiteSaleProductsItem = options.Class.extend({
         $ribbons.removeClass(htmlClasses);
 
         $ribbons.addClass(ribbon.html_class || '');
-        $ribbons.attr('style', `background-color: ${ribbon.bg_color || ''} !important`);
+        $ribbons.attr('style', `background-color: ${ribbon.bg_color ? `${ribbon.bg_color} !important` : 'inherit'}`);
```

Comment:
```suggestion
        $ribbons.attr('style', `background-color: ${ribbon.bg_color ? '${ribbon.bg_color} !important' : 'inherit'}`);
```
use 'single_quote' inside `backtick` for simplicity ;)